livekit-agents/livekit/agents/llm/function_context.py

```diff
@@ -100,6 +102,52 @@ class CalledFunction:
     exception: BaseException | None = None
 
 
+@dataclass
+class FunctionCallsHandle:
+    """Represents a set of function calls to be executed, along with their context"""
+
+    function_calls: Sequence[FunctionCallInfo]
+    llm_stream: LLMStream
+    nest_depth: int
+    speech_id: str
+    _interrupted: bool = False
+
+    async def execute(self) -> list[CalledFunction]:
+        called_fncs = []
```

Comment:
I don't think this should be part of the llm module. It is very specific to the PipelineAgent